gfx/ASCII_Overmap/tile_config.json

```
@@ -1,6 +1,14 @@
 {
   "tile_info": [
-    { "pixelscale": 1, "width": 10, "height": 10, "iso": false, "retract_dist_min": -1.0, "retract_dist_max": 1.0 }
+    {
+      "pixelscale": 1,
```

```suggestion
      "pixelscale": 1
```
Making change to reset tests to non broken state.